packages/api/src/middleware/acl/hasFarmAccess.js

```diff
@@ -13,6 +13,7 @@ const entitiesGetters = {
   expense_type_id: fromFarmExpenseType,
   nitrogen_schedule_id: fromNitrogenSchedule,
   farm_id: (farm_id) => ({ farm_id }),
+  locationIds: fromLocations,
```

Comment:
This is confusing. 
It's better to create another function from fromLocationIds.
Then fromLocation will just call fromLocationIds

Reply:
Yeah it is and I did do that first. 

But I was looking at the code and the only other implementation that uses the locations: fromLocations is unused code I believe from here: https://github.com/LiteFarmOrg/LiteFarm/pull/1920

But I agree with you I did something messy to not repeat similar code. So I will bring back that function.